jgiganto/MorseToLight/readme.md

```diff
@@ -0,0 +1,201 @@
+# MORSE TO LIGHT
+Muy buenas, en esta ocasión vamos a aprender a realizar una app con Xamarin para dispositivos Android. Básicamente la función de nuestra app será traducir un texto a código morse y transmitirlo usando para ello el flash de la cámara de nuestro smartphone.
+Para ello vamos a apoyarnos en la API multiplataforma Xamarin Essentials, la cual permite hacer uso de los sensores del dispositivo (giroscopio, brújula, etc..) de una manera sencilla.
```

Comment:
In Markdown, you must always put a empty line between paragraphs. If not, the text like in this case is concatenated.